.gitlab/ci/global.yml

```diff
@@ -91,6 +91,9 @@
     - |
       if [ -n "${DESTROY_INSTANCES}" ]; then
         python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ARTIFACTS_FOLDER/env_results.json "$INSTANCE_ROLE" "$TIME_TO_LIVE"
+        export PSWD=$(jq .serverLogsZipPassword < $(cat secret_conf_path) | cut -d \" -f 2)
+        zip -P $PSWD -j $ARTIFACTS_FOLDER/Logs.zip $ARTIFACTS_FOLDER/server*.log || ((($? > 0)) && echo "Didn’t find any server logs, skipping this stage" && exit 0)
+        rm -f $ARTIFACTS_FOLDER/server*.log
```

Comment:
I thought it was decided that we didn't need to archive the logs and password protect them because GitLab is access restricted anyways.
```suggestion
```

Reply:
cool didn't know